src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.ServiceConnection.ps1

```diff
@@ -264,6 +264,7 @@ class ServiceConnection: ADOSVTBase
                     $controlResult.AddMessage("Global groups that have access to service connection.",$restrictedGroups)
                     $controlResult.SetStateData("Global groups that have access to service connection",$restrictedGroups)
                     $controlResult.AdditionalInfo += "Count of global groups that have access to service connection: " + @($restrictedGroups).Count;
+                    $controlResult.AdditionalInfoInCSV ="global groups that have access to service connection."
```

Comment:
Global groups.... Also the list of global groups is not mentioned. Seems incomplete?